__tests__/unit.test.ts

```diff
@@ -4,19 +4,33 @@ import CloudWatch from 'aws-sdk/clients/cloudwatch'
 
 jest.mock('aws-sdk/clients/cloudwatch')
 
+const ENVIRONMENT_VARIABLE_OVERRIDES = {
+  GITHUB_REPOSITORY: 'MY-REPOSITORY-NAME',
+  GITHUB_WORKFLOW: 'MY-WORKFLOW-ID',
+  GITHUB_ACTION: 'MY-ACTION-NAME',
+  GITHUB_ACTOR: 'MY-USERNAME[bot]',
+  GITHUB_REF: 'MY-BRANCH',
+  GITHUB_SHA: 'MY-COMMIT-ID',
+}
+
 describe('unit test suite', () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    process.env = {...OLD_ENV, ...ENVIRONMENT_VARIABLE_OVERRIDES};
+  })
 
   test('checkStatusString passes on valid success string', async () => {
```

Comment:
I think that it would make more sense for unit tests to be sequential

Reply:
What do you mean? They are in sequence (based on their order in `cw-build-status.ts`